### ISSUE TITLE
File share does not work if mobile number is not saved

### ISSUE THREAD

**Issue by SandeepJ007** [Amitpatil215/whatsapp_share#21]
When we try to share file to the number which is not saved in contact, it does not open chat thread with the number.

**Comment by amithaak2602** [comment]
any solution ?

**Comment by Jenil-Infyom** [comment]
**Add Flags in shareFile Function into Android Native Side Code as per below Screenshot :- **

![Screenshot 2024-06-04 at 5 29 21 PM](https://github.com/Amitpatil215/whatsapp_share/assets/109334364/faa1e389-f9a2-49c5-b72b-63fa91fb720d)

**Comment by SandeepJ007** [comment]
NO, It does not work event if adding Intent Flag, I tried to add intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK); but it does not work.

**Comment by naislakes** [comment]
Hello, I am still encountering the same issue when using WhatsApp Business
